client/components/deposits-overview/test/index.tsx

```diff
@@ -187,6 +197,22 @@ describe( 'Deposits Overview information', () => {
 		expect( getByText( 'Estimated' ) ).toBeTruthy();
 		expect( getByText( '—' ) ).toBeTruthy();
 	} );
+
+	test( 'Confirm new account waiting period notice does not show', () => {
+		global.wcpaySettings.accountStatus.deposits.completed_waiting_period = true;
+		const { getByText } = render( <DepositsOverview /> );
+		expect(
+			getByText( 'Your first deposit is held for seven business days' )
+		).toBeFalsy();
```

Comment:
This one is failing since `getByText` [will fail the test if it can't find the element](https://testing-library.com/docs/queries/about). `queryByText` will return null, which can be useful for asserting an element is not present.
```suggestion
		const { queryByText } = render( <DepositsOverview /> );
		expect(
			queryByText( 'Your first deposit is held for seven business days' )
		).toBeFalsy();
```